pulsar-discovery-service/src/main/java/org/apache/pulsar/discovery/service/server/ServiceConfig.java

```diff
@@ -34,6 +33,10 @@
  */
 public class ServiceConfig implements PulsarConfiguration {
 
+    // Discovery service doesn't send any messages except Command connected.
+    // So it's ok use a default value.
+    public final static int MAX_MESSAGE_SIZE = 5 * 1024 * 1024;
```

Comment:
Don't need to redefine the same constant. Just reuse `Commands.DEFAULT_MAX_MESSAGE_SIZE`

Reply:
ok